job-scheduler/src/main/java/org/hawkular/metrics/scheduler/impl/SchedulerImpl.java

```diff
@@ -140,7 +140,8 @@
     static final int SCHEDULING_LOCK_TIMEOUT_IN_SEC = 5;
 
     // TODO We probably want this to be configurable
-    static final int JOB_EXECUTION_LOCK_TIMEOUT_IN_SEC = 3600;
+//    static final int JOB_EXECUTION_LOCK_TIMEOUT_IN_SEC = 3600;
```

Comment:
Good catch. Yes, it needs to be reverted. Right now it is hard coded, but that will change with [HWKMETRICS-551](https://issues.jboss.org/browse/HWKMETRICS-551).